dvc/commands/queue/status.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import logging
+from typing import List, Mapping, Optional
+
+from dvc.cli.command import CmdBase
+from dvc.cli.utils import append_doc_link
+from dvc.compare import TabularData
+
+from ..experiments.show import format_time
+
+logger = logging.getLogger(__name__)
+
+
+class CmdQueueStatus(CmdBase):
+    """Kill exp task in queue."""
+
+    def run(self):
+        result: List[
+            Mapping[str, Optional[str]]
+        ] = self.repo.experiments.celery_queue.status()
+        all_headers = ["Rev", "Name", "Created", "Status"]
+        td = TabularData(all_headers)
+        for exp in result:
+            created = format_time(exp.get("timestamp"))
+            td.append(
+                [exp["Rev"], exp.get("Name", ""), created, exp["Status"]]
+            )
+        td.render()
```

Comment:
I think we might want to sort these by timestamp, but that doesn't have to go in this PR (since we'll probably up with different sorting/filtering flags later)